CRM/Odoosync/Sync/Contribution/Data/ContributionParam.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+class CRM_Odoosync_Sync_Contribution_Data_ContributionParam extends CRM_Odoosync_Sync_Contribution_Data {
+
+  /**
+   * Account relationship id
+   *
+   * @var int
+   */
+  const ACCOUNT_RELATIONSHIP_ID = 3;
```

Comment:
what if civicrm changed the this value in the future to be something other than 3 ? I would remove this constant and use the API below to fetch the relationship id using the name which is less likely to change. same applies for other classes below.